Add auto_preview setting for Markdown, SVG, and CSV files (#57010)
zed-industries/zed: docs/src/reference/all-settings.md

`boolean` values

### Auto preview

> - Description: Whether to automatically open a preview when opening previewable files (Markdown, SVG, and CSV).

Reviewer: ```suggestion
- Description: Whether to automatically open a preview when opening files with preview capabilities (e.g. Markdown, SVG, CSV).
```
